pkg/cmd/attestation/verify/verify_test.go

```diff
@@ -333,26 +345,29 @@ func TestRunVerify(t *testing.T) {
 	t.Run("with invalid OIDC issuer", func(t *testing.T) {
 		opts := publicGoodOpts
 		opts.OIDCIssuer = "not-a-real-issuer"
+		opts.SigstoreVerifier = &verification.FailSigstoreVerifier{}
```

Comment:
If we pass in a FailSigstoreVerifier which _always_ fails, what is this test… testing?

Reply:
i.e. my impression was that this is exercised inside the sigstore-go code, which we no longer call with the Fail verifier 🤔

Reply:
Chatted about this test offline, it will be skipped for now and we will add another layer of testing that allows us to more thoroughly test the layer of code that would handle OIDC issuers.